pull_request_template.md

```diff
@@ -0,0 +1,87 @@
+<!--Thank you for contributing to AllenSDK, your work and time will help to
+advance open science! For full contribution guidelines check out our
+guide on GitHub here, https://github.com/AllenInstitute/AllenSDK/blob/master/CONTRIBUTING.md-->
+
+# Overview:
+<!-- Give a brief overview of the issue you are solving. Succinctly
+explain the GitHub issue you are addressing and the underlying problem
+of the ticket
```

Comment:
I would maybe also add that the commit message for the commits addressing the issue should also include this message?

Reply:
Good suggestion, I added a one line reminder to include good commit messages!